application/jaxrs-activemq-quarkus/src/test/java/webapp/tier/service/ActiveMqServiceTest.java

```diff
@@ -18,9 +23,15 @@
 class ActiveMqServiceTest {
 
 	@Inject
-	ActiveMqService svc;
+	private ActiveMqService svc;
 
-	String respbody = "message: Hello k8s-3tier-webapp with quarkus";
+	@Inject
+	private ConnectionFactory connectionFactory;
+
+	@ConfigProperty(name = "activemq.queue.name")
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/yurake/k8s-3tier-webapp/pullRequest?prid=6118916)